lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.ts

```diff
@@ -132,14 +134,17 @@ export class TaskListCloudComponent extends BaseTaskListCloudComponent {
     candidateGroupId: string = '';
 
     constructor(private taskListCloudService: TaskListCloudService,
+                private taskCloudService: TaskCloudService,
                 appConfigService: AppConfigService,
                 userPreferences: UserPreferencesService) {
         super(appConfigService, userPreferences, TaskListCloudComponent.PRESET_KEY);
     }
 
     load(requestNode: TaskQueryCloudRequestModel) {
         this.isLoading = true;
-        this.taskListCloudService.getTaskByRequest(<TaskQueryCloudRequestModel> requestNode).subscribe(
+        this.taskListCloudService.getTaskByRequest(<TaskQueryCloudRequestModel> requestNode).pipe(
```

Comment:
you don't need casting if `getTaskByRequest` returns correct result, or fix the `getTaskByRequest` to return it